Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.py

```diff
@@ -1594,8 +1594,10 @@ def main():
     args: dict = demisto.args()
     params: dict = demisto.params()
     self_deployed: bool = params.get('self_deployed', False)
-    tenant_id: str = params.get('tenant_id', '') or params.get('_tenant_id', '')
-    auth_and_token_url: str = params.get('auth_id', '') or params.get('_auth_id', '')
+    tenant_id: str = params.get('tenant_id', '') or params.get('_tenant_id', '') or (params.get('creds_tenant_id')
```

Comment:
Let's add a comment here saying why we have these different param names (due to the credentials change)